src/backend/nodes/cypher_outfuncs.c

```diff
@@ -216,7 +215,6 @@ void out_cypher_relationship(StringInfo str, const ExtensibleNode *node)
 
     WRITE_STRING_FIELD(name);
     WRITE_STRING_FIELD(label);
-    WRITE_STRING_FIELD(parsed_label);
```

Comment:
Why were these removed? **parsed_label** was added into the struct for variable name lookup.